mithril-client-cli/src/services/snapshot.rs

```diff
@@ -281,19 +278,19 @@ impl SnapshotService for MithrilClientSnapshotService {
             .unwrap()
             .with_key("eta", |state: &ProgressState, w: &mut dyn Write| write!(w, "{:.1}s", state.eta().as_secs_f64()).unwrap())
             .progress_chars("#>-"));
-        self.snapshot_client
-            .download_unpack(
-                &snapshot_entity.artifact,
-                &db_dir,
-                DownloadProgressReporter::new(pb, progress_output_type),
-            )
-            .await
-            .with_context(|| {
-                format!(
-                    "Could not download file in directory '{}'",
-                    download_dir.display()
-                )
-            })?;
+        // self.snapshot_client
```

Comment:
1°Why is this code commented out? Shan't it be removed instead?
2°To comment code blocks properly, use `/* … */` instead

Reply:
This code is outdated.